docs/subnets/create-a-vm-timestampvm.md

```diff
@@ -1251,8 +1251,7 @@ JSON file at `~/.avalanchego/configs/vms/aliases.json` with:
 
 ### Installing a VM
 
-AvalancheGo searches for and registers plugins under the `plugins` directory of the
-[build directory](../nodes/maintain/avalanchego-config-flags#build-directory).
+AvalancheGo searches for and registers plugins under the `plugins` [directory](../nodes/maintain/avalanchego-config-flags#plugins).
```

Comment:
Just a nit
```suggestion
AvalancheGo searches for and registers plugins under the `plugins` [directory](../nodes/maintain/avalanchego-config-flags#--plugin-dir-string).
```

Reply:
add `.md` to the file name as in `../nodes/maintain/avalanchego-config-flags.md#--plugin-dir-string`